sql/core/src/test/resources/sql-tests/results/table-valued-functions.sql.out

```diff
@@ -327,13 +327,14 @@ struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
 {
-  "errorClass" : "WRONG_NUM_ARGS.WITHOUT_SUGGESTION",
-  "sqlState" : "42605",
+  "errorClass" : "DATATYPE_MISMATCH.UNEXPECTED_INPUT_TYPE",
```

Comment:
I feel the previous error is better. Shall we tune the named-parameter framework so that it always checks the num args first?